distsql/distsql.go

```diff
@@ -223,7 +223,7 @@ func Select(ctx context.Context, sctx sessionctx.Context, kvReq *kv.Request, fie
 	if !sctx.GetSessionVars().EnableStreaming {
 		kvReq.Streaming = false
 	}
-	resp := sctx.GetClient().Send(ctx, kvReq)
+	resp := sctx.GetClient().Send(ctx, kvReq, sctx.GetSessionVars().KVVars)
```

Comment:
Why not put it in kvReq ?

Reply:
the variables don't belong to the request.
It's long-term variables, only changed by explicit `SET` statement.